source/_posts/2021-07-30-supervisor-update.markdown

```diff
@@ -0,0 +1,55 @@
+---
+title: "Supervisor Update"
+description: "Upcoming changes to the Supervisor"
+date: 2021-07-30 00:00:00
+date_formatted: "July 30, 2020"
+comments: true
+author: Joakim Sørensen
+author_twitter: ludeeus
+categories:
+   - Announcements
+   - Supervisor
+---
+
+## Renaming snapshot to backup
+
+"Snapshot" is a term that we have been using in the Supervisor since the beginning,
+but it's not very descriptive for those that do not know what it is.
+Over the next few weeks, we will start using "backup"
+as the term instead of in all our software and documentation.
+
+The functionality of it does not change, this is just a rename to make it more understandable.
+
+## Supervised installations
+
+<div class='note'>
+Having a supervisor does not make it a supervised installation, Home Assistant Operating System also has this, the information below does not apply to Home Assistant Operating System.
+</div>
+
+While we try not to break supervised installations, we do have a few things we need to change.
+These adjustments you have to manually apply to your installation. Without these adjustments you will start to see warnings in your logs, and your installation will eventually be marked as unsupported.
+
+If you are interested to make changes required on supervised installations more maintainable, have a look at the [blog on the developer site][dev_blog].
+
+As an alternative to doing these adjustments, you can migrate your installation to [Home Assistant Operating System][installation_docs].
+
+### Environment variables
+
+There are a few environment variables that you have to add in order to make the Supervisor work properly with newer versions of the Supervisor.
```

Comment:
Can we name a version here?

Reply:
Versions do not matter, the supervisor auto-updates and this is already on stable.